### DISCUSSION TITLE
Filtering within a list of source members not working

### DISCUSSION THREAD

**Discussion by yepadoos** [codefori/vscode-ibmi#2025] [Q&A]
Version 2.10.0
ctrl-f within the object browser list of members no longer works to filter the member list.  I tried reverting to 2.9.0 and am still having issues.

Thx in advance for help.

**Comment by thomprl** [comment]
Does the search box open?  Mine seems to be working.  You can click the little icon to switch from Search and Filter.

![image](https://github.com/codefori/vscode-ibmi/assets/17936568/c27aee89-b8c4-484c-b559-d4747acc6598)

**Reply by yepadoos** [reply]
No i do not get that pop up any longer.  I reverted to the March 2024 version of vscode and it is back hopefully a fix may come.

**Comment by worksofliam** [comment]
This is part of VS Code and not Code for IBM i. See my comment here: https://github.com/codefori/vscode-ibmi/issues/2027#issuecomment-2093235305
